modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -507,7 +508,15 @@ public String getOutputDir() {
         return outputFolder();
     }
 
-    public void setTemplateDir(String templateDir) {
+    public String getInputSpec() {
+		return inputSpec;
+	}
```

Comment:
Done !

Alexandre

2016-08-17 12:20 GMT+02:00 wing328 <notifications@github.com>:

> In modules/swagger-codegen/src/main/java/io/swagger/codegen/
> DefaultCodegen.java
> <https://github.com/swagger-api/swagger-codegen/pull/3592#discussion_r75097272>
> :
>
> > @@ -507,7 +508,15 @@ public String getOutputDir() {
> >          return outputFolder();
> >      }
> >
> > -    public void setTemplateDir(String templateDir) {
> > +    public String getInputSpec() {
> > +		return inputSpec;
> > +	}
>
> It seems to me the code is not indented properly (using tab instead of
> spaces). Would you please fix it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/swagger-api/swagger-codegen/pull/3592/files/5c1c9eb0b0ba4c1f8dd88ea67202920a509d17fa#r75097272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUA75pocO2DCoFi3sXfDWV3TqZ-CYQaYks5qguBcgaJpZM4JlKSv>
> .
>